style/components/text-utilities.css

```diff
@@ -0,0 +1,27 @@
+.txt-sm {
+  font-size: 0.8rem;
+  line-height: 1rem;
+  /* margin-top: 0.5rem;
+  margin-bottom: 1rem; */
```

Comment:
These lines can be removed , if it is not needed anymore.

Reply:
Thank you , changed the following .